packages/desktop-client/playwright.config.ts

```diff
@@ -1,7 +1,13 @@
-import { expect, defineConfig } from '@playwright/test';
+import {
+  expect as baseExpect,
+  defineConfig,
+  type Locator,
+} from '@playwright/test';
 
-expect.extend({
-  async toMatchThemeScreenshots(locator) {
+export { test } from '@playwright/test';
+
+export const expect = baseExpect.extend({
```

Comment:
Is there a reason to export instead of the previous method?

Reply:
Is this not changed by the `global.d.ts` ? As it was working before?

Reply:
Good point. Not sure how it was working before. But the playwright guide/docs does it this way: https://playwright.dev/docs/test-assertions#add-custom-matchers-using-expectextend